bigtable/releasing/do-release.sh

```diff
@@ -24,16 +24,6 @@ VERSION=$1
 # Change to root of the repo
 cd "$(dirname "$0")/.."
 
-# GitHub release
-
-# release target requires that the current SHA have a sem-ver tag. That's the version it will use when creating the release.
-$PREFIX git tag bigtable/"$VERSION"
-# make sure GITHUB_TOKEN is exported, for the benefit of this next command
-export GITHUB_TOKEN
-GO111MODULE=on $PREFIX make release
-# if that was successful, it could have touched go.mod and go.sum, so revert those
-$PREFIX git checkout go.mod go.sum
-
```

Comment:
`goreleaser` is not happy with prefixed tags,since they are not strictly in semantic version format.
It seems to be a Pro version feature: https://goreleaser.com/customization/monorepo/

While we figure out a solution (forking it, or using something else), I think it's better to remove this step from the release.